test/javascript-wrapper-mocha.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * File: javascript-wrapper.js
+ * Unit tests for the APIs defined in components/javascript-wrapper.js
+ */
+
+var chai = require('chai');
+
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+var expect = chai.expect;
+var should = chai.should();
+
+var sinon = require('sinon');
+
+var _ = require('underscore');
+var noflo = require('noflo');
+
+var jswrapper = require('../components/javascript-wrapper');
+var commonTest = require('./common-test');
+
+describe('javascript-wrapper', function() {
```

Comment:
Can you add a test to ensure the updater's this context is called with the nodeInstance?